### DISCUSSION TITLE
question about streaming playlists

### DISCUSSION THREAD

**Discussion by mrSaT13** [navidrome/navidrome#3091] [Ideas]
tell me if there is a possibility of creating a dynamic streaming playlist, that is, I turn on a song, for example, I click on the stream by track button and navidrom creates a streaming playlist based on similar tracks. until I want to change this by switching the track ?

**Comment by deluan** [comment]
Navidrome UI does not have this functionality yet, but you can do this in some Subsonic clients. Ex for an "Artist Radio" in DSub:
![Screenshot 2024-06-13 at 7 36 32 PM](https://github.com/navidrome/navidrome/assets/331353/5f27ae31-6750-40ea-83e8-d2d7fd969ade)

Symfonium also has this feature. Probably other clients as well.

**Comment by mrSaT13** [comment]
will you be planning?

**Comment by mrSaT13** [comment]
Many thanks to you for your help and time spent.
